Entities/Characters/Scripts/RunnerKnock.as

```diff
@@ -67,19 +67,19 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 	{
 		//get sponge
 		CBlob@ sponge = null;
-		//find the sponge with lowest absorbed
+		//find the sponge with highest absorbed
 		CInventory@ inv = this.getInventory();
 		if (inv !is null)
 		{
-			u8 lowest_absorbed = 100;
+			int8 highest_absorbed = -1;
```

Comment:
(this code is vulnerable to overflows, though they shouldn't happen; will be fixing it in post - don't bother using int8 in most circumstances, just use int. u8 is just used to optimise the property size on net, so using a matching type for the search made sense; using a matching bit width but signed type does not)